AutomationApp/LoggerCallbackImpl.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+
+namespace AutomationApp
+{
+    class LoggerCallbackImpl : ILoggerCallback
+    {
+        private readonly StringBuilder _logCollector = new StringBuilder();
+
+        public void Log(Logger.LogLevel level, string message, bool containsPii)
```

Comment:
if you have "using Microsoft.Identity.Client" then you should have no need to write Logger.LogLevel. you cans imply reference LogLevel